boto3/docs.py

```diff
@@ -591,11 +592,122 @@ def document_operation(operation_model, service_name, operation_name=None,
         if key in ignore_params:
             continue
         param_type = py_type_name(value.type_name)
+
+        # Convert the description from HTML to RST (to later be converted
+        # into HTML... don't ask). If the parameter is a nested structure
+        # then we also describe its members.
+        param_desc = html_to_rst(
```

Comment:
This was just moved from down below. It was moved up and a comment added so that we can modify it before rendering. I can add a test and plan to cover the whole module eventually.